source/aggregation-pipeline-builder.txt

```diff
@@ -89,69 +89,211 @@ Limitations
+               :manual:`$match </reference/operator/aggregation/match/>` stage 
+               and only includes the 
+               :manual:`$project </reference/operator/aggregation/project/>`
+               stage:
+               
+               .. figure:: /images/compass/agg-builder-exclude-stage-example.png 
+                  :alt: Aggregation Builder exclude stage example
+
+         .. step:: Click :guilabel:`Run` at the top right of the pipeline 
+            builder. |compass-short| returns your results in the document view. 
+            
+            .. warning:: 
+               
+               When you click :guilabel:`Run`, the aggregation runs against the 
+               entire collection.
+
+         .. step:: To edit the pipeline, click :guilabel:`Edit` at the top of 
+            your pipeline toolbar to return to the aggregation pipeline builder
+            and make changes to your pipeline. 
+
+   .. tab:: Focus Mode 
+      :tabid: focus 
+
+      .. procedure::
+         :style: normal
+         
+         .. step:: In the aggregation pipeline pane, ensure the 
+            :guilabel:`{} Stages` toggle switch is selected.
+
+            .. figure:: /images/compass/agg-builder-stageview-toggle-on.png
+               :alt: Toggle on stage view mode 
+
+         .. step:: At the bottom of the aggregation pipeline pane, select the 
+            :guilabel:`+ Add Stage` button to open a new aggregation stage 
+            modal.
+
+         .. step:: On the upper-right corner of the stage modal, select the 
+            Focus Mode icon to open Focus Mode.
+
+            .. figure:: /images/compass/focus-mode-button.png 
+               :alt: Select the Focus Mode button
+         
+         .. step:: Click the :guilabel:`Select` dropdown and select the 
+            :manual:`aggregation pipeline stage 
+            </reference/operator/aggregation-pipeline/>` to use for the first 
+            stage of the pipeline. 
+
+         .. step:: Fill in your selected pipeline stage. 
+
+            You can adjust the width of the :guilabel:`Stage Input`, pipeline 
+            stage, and the :guilabel:`Stage Output` by dragging their border to 
+            the desired size.
+
+         .. step:: Click the :guilabel:`Add Stage` dropdown to add additional 
+            aggregation stages before or after your last aggregation stage. 
+            Repeat steps 1 and 2 for each additional stage.
+
+            You can add stages with the following keyboard shortcuts: 
+
+            - To add a stage after the current stage: 
+
+              - Windows / Linux: :guilabel:`Ctrl + Shift + A`
+
+              - Mac: :guilabel:`⌘ + Shift + A`
+            
+            - To add a stage before the current stage: 
+              
+              - Windows / Linux: :guilabel:`Ctrl + Shift + B`
+
+              - Mac: :guilabel:`⌘ + Shift + B` 
+
+         .. step::  **(Optional)** Toggle the previews for each stage to include 
```

Comment:
[question]

I can't see what this looks like in the UI, but "previews" strikes me as kind of an odd word choice here. Maybe rather than making this it's own step, this could be a note / tip that says you can enable or disable stages with the toggle switches?

Reply:
Good point — I'll try adding this as a note under the "Fill in your pipeline stage" step.

Reply:
Update: Renders alright!